assets/large_language_models/prompts/generate_a_discharge_letter/spec.yaml

```diff
@@ -0,0 +1,14 @@
+$schema: https://azuremlschemas.azureedge.net/latest/asset.prompt.schema.json
+
+type: prompt
+name: generateadischargeletter
+version: 0.1.0
+display_name: generate_a_discharge_letter
```

Comment:
for display name, i don't think you need underscore anymore.

Reply:
and let's use something more human friendly, such as Generate a Discharge Letter

Reply:
fixed